app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2><%= @mechanic.name %></h2>
+<p>Years of Experience: <%= @mechanic.years_experience %></p>
+<h3>Currently working on these rides:</h3>
+
+<% @mechanic.rides.each do |ride| %>
+  <li><%= ride.name %></li>
+<% end %>
+
+<div id="add_ride">
+<h2>Add a ride to <%= @mechanic.name %>'s workload: </h2>
+<%= form_with url: "/mechanics/#{@mechanic.id}", method: :post do |f| %>
```

Comment:
This form setup looks great! I think a slightly more RESTful path here could be "/mechanics/:id/rides" to indicate that the POST is building the relationship between mechanic and a ride.